UI/js-src/lsmb/PublishSelect.js

```diff
@@ -17,6 +17,11 @@ define(["dojo/_base/declare",
                               self.publish(targetValue);
                           })
                    );
+               },
+               startup: function() {
```

Comment:
Why do you need this change? Most widgets don't need a startup function and I'm not seeing anything happen here which the regular startup function isn't already doing...

Reply:
Dojo decided that a transition from undefined to defined should not be published. Here we have dependent fields which clearly need to stay synced. It must be in startup because we rely on postCreate to initialize fields, so we have to wait for them to be properly setup before triggering an update.

Reply:
Actually, Dojo's API documentation explicitly states that the locale value can *only* be set before Dojo-boot (at page load time). So, I think this approach is asking for trouble down the line: not a single widget will detect the change.

However, none of that explains why we need a "startup" function? The function is only called when the widget is instantiated, after all...

Reply:
I do not change locale. I display a value dependent on another. I a country is selected, then the date placeholder has to change accordingly.